### DISCUSSION TITLE
Gradle nested project's *.versions.toml file is not processed

### DISCUSSION THREAD

**Discussion by roded** [renovatebot/renovate#20622] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'd like Renovate to pickup Gradle version TOML files in nested project directories.

### What you tried so far.

I have the following directory structure in a Gradle project:
```
buildlibs.versions.toml
/nested-project/projectlibs.versions.toml
```

And the following renovate.json:
```
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "extends": [
        "config:base"
    ],
    "prConcurrentLimit": 2,
    "prHourlyLimit": 3,
    "baseBranches": [
        "develop"
    ],
    "enabledManagers": [
        "gradle",
    ],
    "includePaths": [
        "buildlibs.versions.toml",
        "nested-project/**"
    ],
    "packageRules": [
        {
            "matchPaths": [
                "nested-project/**"
            ],
            "additionalBranchPrefix": "nested-project/"
        },
    ]
}
```

Renovate picks up `buildlibs.versions.toml` but not `/nested-project/projectlibs.versions.toml`.
I'm expecting it to pick up `/nested-project/projectlibs.versions.toml` since `nested-project/**` is included in `includePaths` and the Gradle managers default file matching array includes `\.versions\.toml$` (https://docs.renovatebot.com/modules/manager/gradle/). Are any of my assumptions wrong?

Second, is it possible to debug Renovate's file matching when using it via the GitHub bot?

Thanks

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by Churro** [comment, marked as answer]
Have you tried without `includePaths` or other custom settings in `renovate.json`?

For more insights, please provide a minimum reproduction repo.

**Reply by roded** [reply]
Is my only option in debugging this merging such a change into the repository and waiting to see what renovate will do?
I'd rather avoid that, it'll create quite a lot of noise in the repository.
Is not possible to enable debugging of file matching in renovate?
Thanks

**Reply by roded** [reply]
Found a type on the `includePaths`!
Sorry for the trouble. I debugging option would've help though.
Thanks!

**Reply by Churro** [reply]
One option could be to fork the repo and then do this change.

Please have a look here on how to retrieve logs: https://docs.renovatebot.com/troubleshooting/
